src/renderer/Presentational/AddEthereumNode/AddEthereumNode.tsx

```diff
@@ -151,30 +144,43 @@ AddEthereumNodeProps) => {
     sNodeStorageLocationFreeStorageGBs,
     setNodeStorageLocationFreeStorageGBs,
   ] = useState<number>();
-  const [sHasSeenAlphaModal, setHasSeenAlphaModal] = useState<boolean>();
```

Comment:
alpha modal logic is moved to `AddNode` (the first step after seeing splash screen)